seacrowd/sea_datasets/thai_hh_rlhf/thai_hh_rlhf.py

```diff
@@ -0,0 +1,137 @@
+  title={Training a helpful and harmless assistant with reinforcement learning from human feedback},
+  author={Bai, Yuntao and Jones, Andy and Ndousse, Kamal and Askell, Amanda and Chen, Anna and DasSarma, Nova and Drain, Dawn and Fort, Stanislav and Ganguli, Deep and Henighan, Tom and others},
+  journal={arXiv preprint arXiv:2204.05862},
+  year={2022}
+}
+
+"""
+
+_LOCAL = False
+_LANGUAGES = ["tha"]
+_DATASETNAME = "thai_hh_rlhf"
+_DESCRIPTION = """\
+This is a Thai-translated dataset based on Anthropic/hh-rlhf using Google Cloud Translation. This repository 
+provides access to 161K Train dataset Anthropic/hh-rlhf (Thai-translated). 
+
+The original repository provides access to two different kinds of data:
+
+1. Human preference data about helpfulness and harmlessness from Training a Helpful and Harmless Assistant with 
+Reinforcement Learning from Human Feedback. These data are meant to train preference (or reward) models for 
+subsequent RLHF training. These data are not meant for supervised training of dialogue agents. Training dialogue 
+agents on these data is likely to lead to harmful models and this shold be avoided.
+
+2. Human-generated and annotated red teaming dialogues from Red Teaming Language Models to Reduce Harms: Methods, 
+Scaling Behaviors, and Lessons Learned. These data are meant to understand how crowdworkers red team models and 
+what types of red team attacks are succesful or not. The data are not meant for fine-tuning or preference modeling 
+(use the data above for preference modeling). These data are entire transcripts of conversations that are derived 
+from the harmlessness preference modeling data described above, where only the chosen response is incorporated into 
+the overall transcript. Furthermore, the transcripts are annotated with human and automated measurements of how 
+harmful the overall dialogues are.
+
+The translated data corresponds to the first kind of data.
+"""
+
+_HOMEPAGE = "	https://huggingface.co/datasets/Thaweewat/hh-rlhf-th"
+_LICENSE = Licenses.MIT.value
+_URL = "https://huggingface.co/datasets/Thaweewat/hh-rlhf-th/resolve/main/hh-rlhf-train-161k-th.parquet"
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ThaiDatabricksDollyDataset(datasets.GeneratorBasedBuilder):
+    """Thai Alpaca Dataset"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "qa"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source", 
+            version=SOURCE_VERSION, 
+            description=f"Thai-Alpaca source schema", 
+            schema="source", 
+            subset_id=_DATASETNAME,
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"Thai-Alpaca SEACrowd schema",
```

Comment:
Same as line 68, only for this line, since there are no variable references, you may remove the f. Also, the description should be Thai_HH_RLHF SEACrowd schema